utils/docker/Dockerfile.centos.7

```
@@ -164,10 +150,15 @@ COPY utils/sl utils/sl
 COPY utils/build.config utils/
 RUN chown -R daos_server.daos_server /home/daos
 USER daos_server:daos_server
-RUN ln -s utils/sl site_scons
 
-RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || \
-    (scons --build-deps=yes install PREFIX=/opt/daos --deps-only && \
+# Control what to build.  By default Dockerfiles build everything to allow for
+# ease-of-use for users, however in CI everything is turned off and then
+# selectively enabled.  Turning off any step disables all future steps.
+ARG DAOS_DEPS_BUILD=yes
+ARG DEPS_JOBS=1
+
+RUN ln -s utils/sl site_scons && [ "$DAOS_DEPS_BUILD" != "yes" ] || \
+    (scons --build-deps=yes --jobs $DEPS_JOBS PREFIX=/opt/daos --deps-only && \
     /bin/rm -rf build *.gz)
```

I won't repeat for every instance here, but for simply grouping commands like you are doing here, braces (`{}`) are preferred to brackets (`()`) since the latter actually invokes the pipeline in a subshell, which incurs a `fork()`/`exec()`:
```suggestion
    { scons --build-deps=yes --jobs $DEPS_JOBS PREFIX=/opt/daos --deps-only && \
    /bin/rm -rf build *.gz; }
```

Note the trailing `;` before the closing brace (`}`).